### DISCUSSION TITLE
xdrip no longer uploading to nightscout

### DISCUSSION THREAD

**Discussion by gitanO0** [NightscoutFoundation/xDrip#2838] [General]
This has been working flawlessly for a couple of years+ but tonight about an hour ago my readings are no longer being sent to my nightscout site for some reason.  In xdrip+ I'm getting an error "Unable to do REST API Upload:  Service Unavailable marking record:  failed".    I've gone to https://xxx.herokuapp.com/api/v1/status.json and everything looks good on that status page and https://xxx.herokuapp.com/api/v1/entries is showing the last 10 entries up to about an hour ago.  

What might be causing xdrip+ not to be able to talk to my nightscout site?

**Comment by Navid200** [comment]
Full database?

**Reply by gitanO0** [reply]
It's showing about 180mb of the 500mb filled.  I did a "delete entries before 180 days ago" and I'm now down to 160mb

**Reply by Navid200** [reply]
I am seeing some reports of issues with Atlas.  You are not alone.
I'm afraid we have to wait for @psonnera to come online and advise.  He must be sleep now.

**Comment by gitanO0** [comment]
Ah, ok, so just sit tight.  I am seeing right when things went south there was a big "write" operation and then nothing...

**Comment by gitanO0** [comment]
oop... and just that, it's back up.  Odd.

**Comment by gitanO0** [comment]
I guess it's all good now.

**Reply by YOUSIFALHAJ** [reply]
Hi ., I have the same issue can you help with how it has been solved

**Reply by Navid200** [reply]
There are different possible causes of such a problem.
Why don't you open a new thread?  I will help you solve it right away.

There is no reason to bundle all the requests for help related to uploads to Nightscout.  No matter why the server stops, you will not be able to upload to Nightscout.  That does not mean all the problems have the same requirement to solve.

**Reply by Navid200** [reply]
Just go to discussions on the following link and click on this:
![Screenshot 2023-05-26 093126](https://github.com/NightscoutFoundation/xDrip/assets/51497406/7ce45879-1997-4720-90d2-9cf220334f8c)  
  
  
https://github.com/NightscoutFoundation/xDrip/discussions
